forem/forem
# fix gists styling adding width to tds

## What type of PR is this? (check all applicable)


## Description

Fixes broken styling in Chrome & Edge for embed Gists.

## Related Tickets & Documents



- Related Issue # https://github.com/forem/forem/issues/18893
- Closes # https://github.com/forem/forem/issues/18893

## QA Instructions, Screenshots, Recordings

Confirm that gists take full width in any device mobile/desktop. Just pick any thread with any gist embed [eg](https://dev.to/qarunqb/tdd-in-angular-dependency-injection-and-mocking-4jnh)) and see how the gists are again displayed correctly.

### UI accessibility concerns?

From this

![broken gist desktop](https://user-images.githubusercontent.com/68973307/211220363-fc588627-bd16-4838-b803-d9ab89bda063.png)

To this

![fixed gists desktop](https://user-images.githubusercontent.com/68973307/211220387-ea1a08d2-85d7-4e55-a1b6-dafde39da9fc.png)

![fixed gists mobile](https://user-images.githubusercontent.com/68973307/211220396-ecd172c3-8e71-4f8e-9a3d-de832373374a.png)

## Added/updated tests?

- [x] No, and this is why: simple css change

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
